cmake/UseCyclus.cmake

```diff
@@ -89,11 +89,15 @@ MACRO(USE_CYCLUS lib_root src_root)
 
 
     # set cpp path
-    IF("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang")
-        SET(PREPROCESSOR "--cpp-path=clang++")
+    
+    IF(DEFINED ENV{CPP})
```

Comment:
add a comment to explain what's happening here

Reply:
@gonuke does it look better ?

Reply:
lgtm